### ISSUE TITLE
Advices on hyperparameter tuning / interactions between hyperparameters

### ISSUE THREAD

**Issue by RemiPasserotti** [emilienschultz/activetigger#352]
For instance, that you have to adjust learning rate and/or weight decay while increasing the batch size

**Comment by emilienschultz** [comment]
You are right. We are going to start writing a cookbook to add it in the documentation in a few days, with @jboelaert and @julesbrion to specifically tackle this question (advices on how to improve/check model fine-tuning).

I will let this issue open, since we can start by adding elements here if needed.

**Comment by emilienschultz** [comment]
I open a specific issue with the result of the meeting, that Annina will integrate in the documentation
